src/Microsoft.DotNet.Build.Tasks/UpdateProjectDependencies.cs

```diff
@@ -0,0 +1,146 @@
+﻿using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NuGet.Versioning;
+using System;
+using System.IO;
+using System.Linq;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public class UpdateProjectDependencies : Task
+    {
+        [Required]
+        public ITaskItem ProjectJson { get; set; }
+
+        public string OldPrerelease { get; set; }
```

Comment:
I'm going to split update into two tasks/targets--it really is two separate commands anyway. That'll let me get rid of the boolean xor, too.